src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -321,13 +328,35 @@ function handleAddChart(layoutInfo, chartId) {
  * @param plotId
  * @returns {*}
  */
-function handlePlotActive(layoutInfo, plotId) {
+function handlePlotActive(layoutInfo, action) {
+    const plotId = action.payload.plotId;
     const tableId = get(layoutInfo, ['images', 'activeTableId']);
 
     var rowNum= Number(plotId.substring(plotIdRoot.length));
     dispatchTableHighlight(tableId, rowNum);
     return layoutInfo;
 }
+
+/**
+ * handle logic when a new search is initiated.
+ * @param {LayoutInfo} layoutInfo layoutInfo
+ * @param {string} action
+ * @returns {LayoutInfo} the new layoutInfo
+ */
+function handleNewSearch(layoutInfo, action) {
+    const tbl_id = get(action, 'payload.request.META_INFO.tbl_id');
+    if (tbl_id === LC.RAW_TABLE) {
+        removeTablesFromGroup();
+        removeTablesFromGroup(LC.PERIODOGRAM_GROUP);
```

Comment:
> tgoldina: ... how do we keep raw table in sync with the related tables? If user filters Raw and Phase Folded tables separately, we can not synchronize the highlight index between them, can we?

You are right.  I think a filter of a raw table should trigger a reset condition.

hello